package.json

```diff
@@ -24,41 +24,41 @@
         "start-electron": "electron build/electron/index.js",
         "fix-locale-files": "node ./app/assets/locales-fixup.js",
         "create-notification-string": "node ./app/assets/create-notifcation-string.js",
         "remove-english-locale-duplicates": "node ./app/assets/locales-remove-english-duplicates.js",
         "postfix-locale-files": "./node_modules/prettier/bin-prettier.js app/assets/locales/locale-*.json --write",
-        "build": "cross-env NODE_ENV=production webpack --env.prod",
-        "build-nougly": "cross-env NODE_ENV=production webpack --env.prod --env.noUgly",
-        "build-hash": "cross-env NODE_ENV=production webpack --env.prod --env.hash --env.baseUrl=''",
-        "build-hash-nougly": "cross-env NODE_ENV=production webpack --env.prod --env.hash --env.noUgly --env.baseUrl=''",
-        "build-github": "cross-env NODE_ENV=production webpack --env.prod --env.hash --env.deprecated --env.baseUrl='/wallet/'",
-        "build-electron": "cross-env NODE_ENV=production webpack --env.prod --env.electron --env.hash",
-        "build-electron-nougly": "cross-env NODE_ENV=production webpack --env.prod --env.electron --env.noUgly --env.hash --progress --display-error-details --colors",
+        "build": "cross-env NODE_ENV=production webpack --env prod",
+        "build-nougly": "cross-env NODE_ENV=production webpack --env prod --env noUgly",
+        "build-hash": "cross-env NODE_ENV=production webpack --env prod --env hash --env baseUrl=false",
+        "build-hash-nougly": "cross-env NODE_ENV=production webpack --env prod --env hash --env noUgly --env baseUrl=false",
+        "build-github": "cross-env NODE_ENV=production webpack --env prod --env hash --env.deprecated --env baseUrl='/wallet/'",
+        "build-electron": "cross-env NODE_ENV=production webpack --env prod --env electron --env hash",
+        "build-electron-nougly": "cross-env NODE_ENV=production webpack --env prod --env electron --env noUgly --env hash --progress --display-error-details --colors",
         "prepare-electron": "copyfiles -f resources/*.js build/electron/",
         "postbuild-electron": "npm run prepare-electron",
         "postbuild-electron-nougly": "npm run prepare-electron",
-        "build-win32": "cross-env NODE_ENV=production && webpack --env.prod --progress --colors &&  echo 'copying to electron/build..' &&  xcopy dist ..\\electron\\build /s /e /y && echo 'done.'",
+        "build-win32": "cross-env NODE_ENV=production && webpack --env prod --progress --colors &&  echo 'copying to electron/build..' &&  xcopy dist ..\\electron\\build /s /e /y && echo 'done.'",
         "precommit": "pretty-quick --staged",
         "prepackage": "npm run build-electron",
         "prepackage-no-ugly": "npm run build-electron-nougly",
-        "package": "build -p never",
+        "package": "electron-builder -p never",
```

Comment:
build create the dist to serve through a webserver, electron-builder creates a build for electron. I suggest to not change existing behavior

Reply:
I found that build is not available in the latest version of electron

Reply:
Currently using "build" will cause the compilation to fail. If you know why, you can tell me and I will update it.

Reply:
https://github.com/bitshares/bitshares-ui/issues/3420